Docs/user_guide/modules/brainsfit.md

```diff
@@ -10,7 +10,7 @@ Most frequently used for these scenarios and recommended registration settings.
 
 ### Same Subject: Longitudinal
 
-For this case we're registering a baseline T1 scan with a follow-up T1 scan on the same subject a year later.  The two images are available on the [Midas site](http://midas.kitware.com/item/view/483) as test.nii.gz and test2.nii.gz
+For this case we're registering a baseline T1 scan with a follow-up T1 scan on the same subject a year later.
```

Comment:
@hjmjohnson Is there an authoritative home for BRAINSTool testing data ? 

I found the file [SimpleITK-Notebooks/blob/master/Data/brains_manifest.json](https://github.com/InsightSoftwareConsortium/SimpleITK-Notebooks/blob/master/Data/brains_manifest.json) ... but looks like the collection on data.kitware.com is not public, data can only be referenced using the api.

For example, here is a summary for accessing `test.nii.gz`

| Status  | URL |
|--|--|
| :heavy_check_mark: | https://data.kitware.com/api/v1/file/hashsum/SHA512/038fa4ba7e74233e35052d87d0e525b2f66b78c0b43267b6ab29afffa7306ba31df69ab0445f6d6b5d04dc21a2eb599297d6387eae5fb452ab4189fbb13aa3a6/download | 
| :x: | https://data.kitware.com/#item/5bf963748d777f2179bc07bf | 
 
Last, should I look into migrating the following data (only accessible internally at Kitware):

![image](https://user-images.githubusercontent.com/219043/170889155-d51a6445-5da4-4066-849d-db6922a966d2.png)